### ISSUE TITLE
Links need underlines for web accessibility

### ISSUE THREAD

**Issue by dylanhthomas** [pages-themes/primer#44]
the default a style contains "text-decoration: none"

In this case it fails WCAG 2.0, but more importantly, people with vision issues cannot discern between links within body text.

Before:
![image](https://user-images.githubusercontent.com/824168/93508029-f7fc6780-f8e3-11ea-8800-9991236f88b1.png)

Before without Color:
![image](https://user-images.githubusercontent.com/824168/93508633-3e51c680-f8e4-11ea-9b7d-fa5ea244b567.png)

After:
![image](https://user-images.githubusercontent.com/824168/93508200-021e6600-f8e4-11ea-860a-bcbfcb7f323f.png)

After without Color:
![image](https://user-images.githubusercontent.com/824168/93508659-49a4f200-f8e4-11ea-8b93-bef3d812f547.png)

**Comment by salif** [comment]
I think headers should have visible anchor icons by default instead of underlines
